src/lib/edge-functions/registry.mjs

```diff
@@ -513,7 +548,9 @@ export class EdgeFunctionsRegistry {
    * @returns {Promise<void>}
    */
   async #setupWatcherForDirectory(directory) {
+    const ignored = [new RegExp(this.#servePath)]
```

Comment:
This might match too many files, if `servePath` contains a `.` character or other Regex control sequences. You never know :D
Since this ends up being passed into chokidar, I think this should work better:
```suggestion
    const ignored = [`${this.#servePath}/**`]
```

(see [anymatch docs](https://github.com/micromatch/anymatch#usage) for reference)